test/commands/hset.js

```diff
@@ -24,4 +24,20 @@ describe('hset', () => {
           email: 'bruce@wayne.enterprises',
         })
       ));
+
+  it('should let you set multiple hash map keys and values in a single command', () => {
+    const hash = { id: '2', email: 'bruce@wayne.enterprises' };
+    return redis
+      .hset('user:2', 'id', '2', 'email', 'bruce@wayne.enterprises')
+      .then(status => expect(status).toBe(2))
+      .then(() => expect(redis.data.get('user:2')).toEqual(hash));
+  });
+
+  it('should let you set multiple hash map keys and values with an object', () => {
+    const hash = { id: '3', email: 'bruce@wayne.enterprises' };
+    return redis
+      .hset('user:3', hash)
+      .then(status => expect(status).toBe(1))
+      .then(() => expect(redis.data.get('user:3')).toEqual(hash));
+  });
```

Comment:
return redis
    .hset('user:3', 'id', 3, 'email', 'bruce@wayne.enterprises')   <=== this should be additional arguments like this (field1,val1,field2,val2) etc...        (or using a spread operator and an array)

a subsequent call to redis.hgetall('user:3')  should return all data